ansible/group_vars/omero-hosts.yml

```diff
@@ -11,10 +11,10 @@
 idr_omero_server_release: 5.6.0
 omero_server_checkupgrade_comparator: '!='
 
-idr_omero_web_release: 5.6.3
+idr_omero_web_release: 5.7.0
 # omero-web depends on omero-py but may not pin the latest release
-omero_web_python_addons:
-  - omero-py==5.7.1
+# omero_web_python_addons:
+#   - omero-py==5.7.1
```

Comment:
Is there a reason for commenting this out?

Reply:
No specific reason, this might come from debugging (as bumping `iviewer` alone was not sufficient since upstream `omero-web` versions are required.
I assume we want to keep tracking the individual `omero-web/omero-py` versions in the deployment rather than letting `omero-web` pull the latest compatible dependency on deployment?

Reply:
Yes, that's my preference until we have a [better way of describing dependencies](https://github.com/ome/design/issues/103).

Reply:
Restored in  230267c